controllers/storagecluster/external_resources.go

```diff
@@ -268,7 +269,16 @@ func (obj *ocsExternalResources) ensureCreated(r *StorageClusterReconciler, inst
 			}
 			externalOCSResources[instance.UID] = externalConfig
 		}
+
 		externalClusterClient.Close()
+
+		if err := r.createDefaultClaimFor410UpgradeSCs(instance); err != nil {
```

Comment:
The name does not describe the secanrio, can we change it to:
```suggestion
		if err := r.createClaimsFor410DefaultStorageClasses(instance); err != nil {
```

Reply:
done